mlsshared.sty

```diff
@@ -185,7 +188,7 @@
   breaklines=true,                 % automatic line breaking only at white-space
   keepspaces,                      % don't remove space such as those after closing parenthesis
   captionpos=b,                    % sets the caption-position to bottom
-  commentstyle=\color{commentcolor}\sffamily, % comment style
+  commentstyle=\color{commentcolor}\sffamily\smallifpdf, % comment style
```

Comment:
As far as I can tell this `\smallifpdf` makes no difference, and I would actually have been surprised if the `\smallifpdf` from the `basicstyle` wasn't applied also to comments.  Where exactly do you see a difference?

Reply:
I investigated further and it is messier. Without any style change the LaTeXML result looked like this:

![Untitled](https://user-images.githubusercontent.com/13217430/142411161-20f47944-a1f1-4ca7-b599-4f580378511d.png)

I don't like that the comment-text is larger, and thus I changed to plain \small, and now it looks ok in both acrobat reader (pdf left) and Firefox locally (html right)

![Untitled2](https://user-images.githubusercontent.com/13217430/142411557-416745c6-d05d-48ab-a0ca-d4be8fdebd13.png)

It may be that sans-serif and teletype fonts differ in other ways on other platforms, if so the solution is to avoid sans-serif altogether in code.

Reply:
To me, html looks good with the latest version. But the pdf comments look odd. "-" in "-4" in the table in the comment above looks twice as wide as "-" in "-2" in black on the same line. Also compare the spaces before ";", ",", and "." in the example above and the one under it:
![image](https://user-images.githubusercontent.com/20460714/142431350-80a749d2-dadd-457d-bce5-20d1ba0f175b.png)
It looks almost like there is an extra space in from of "," and ".". Actually, it looks like that without the changes in this PR. So maybe it's irrelevant.

Reply:
It's probably due to the work-around with using hspace{0pt} - that likely could be removed now.

Reply:
(Separately the work-around with hspace{0pt} in that line could be removed, but that doesn't change the comments.)

Reply:
> I investigated further and it is messier. Without any style change the LaTeXML result looked like this:

But the funny thing is that the sizing command you added, `\smallifpdf` is the same as `\normalsize` in the LaTeXML build.  I don't understand.

However, if controlling the size of comments in the LaTeXML build is what we want, then I suppose tweaking the CSS is a much better option?

Reply:
Duplicated conversation.